web/src/constants/ui-text.ts

```diff
@@ -18,3 +18,4 @@ export const UI_TEXT_UPGRADE = "UPGRADE";
 // piped
 export const UI_TEXT_ADD_NEW_KEY = "Add new Key";
 export const UI_TEXT_DELETE_OLD_KEY = "Delete old Key";
+export const UI_TEXT_OPEN_PIPED_CONFIG = "Open piped config";
```

Comment:
How about "View the configuration"

Reply:
![image](https://user-images.githubusercontent.com/26561120/170906017-4d665b38-13b4-4f7f-8c0b-e695a1f901bf.png)

I fixed it (also I expanded the width of the menu).